components/brave_wallet_ui/components/desktop/wallet-onboarding/backup/index.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react'
+
+import {
+  StyledWrapper,
+  Title,
+  Description,
+  IconBackground,
+  PageIcon,
+  TermsRow
+} from './style'
+import { NavButton } from '../../../extension'
+import locale from '../../../../constants/locale'
+import { Checkbox } from 'brave-ui'
+
+export interface Props {
+  onSubmit: () => void
+  backupTerms: boolean
```

Comment:
nit: `isBackupTermsAccepted`

Reply:
> Completely up to you if you want to address any of my nitpicking!

I don't mind the nit's, helps me to be a better dev!! :)

Updated!